tests/test_cli.py

```diff
@@ -260,6 +260,20 @@ def assert_status_down():
         assert result.exit_code == 0
         assert_status_up()
 
+        # test the warning message of image not the latest is not raised
+        assert "Warning!" not in result.output.strip()
```

Comment:
Hmm, you could maybe have this assert earlier after the container is first started. Then you could move the monkeypatch above as well and avoid starting up the container for the third time.